pkg/service/model_build_load_balancer.go

```diff
@@ -487,6 +501,10 @@ func (t *defaultModelBuildTask) buildLoadBalancerName(_ context.Context, scheme
 	_, _ = uuidHash.Write([]byte(scheme))
 	uuid := hex.EncodeToString(uuidHash.Sum(nil))
 
+	if t.service.Annotations[LoadBalancerStackKey] != "" {
+		return fmt.Sprintf("k8s-%.8s", t.service.Annotations[LoadBalancerStackKey]), nil
```

Comment:
This format is used in at least a couple different places. Can it be made a function?

Reply:
done